content/en/serverless/guide/extension_motivation.md

```diff
@@ -0,0 +1,54 @@
+
+### Differences in functionality
+
+{{< img src="serverless/serverless_monitoring_installation_instructions.png" alt="Instrument AWS Serverless Applications"  style="width:100%;">}}
+
+The Lambda extension only supports sending Lambda function telemetry for specific runtimes. The Forwarder is required to collect and add metadata to other AWS service logs, including those from API Gateway, AppSync, Lambda@Edge, etc.
+
+| Runtime | Extension support | Forwarder support |
+| ------- | ----------------- | ----------------- |
+| Python  | Yes               | Yes               |
+| Node.js | Yes               | Yes               |
+| Go      |                   | Yes               |
+| Java    |                   | Yes               |
+| .NET    |                   |                   |
+| Ruby    |                   |                   |
+| PHP     |                   |                   |
+
+For Python and Node.js runtimes, the Lambda extension has full feature parity with the Forwarder. The Lambda extension can submit Lambda logs, Lambda traces (generated by `dd-trace`, not X-Ray), real-time enhanced Lambda metrics, and custom metrics generated by Lambda functions.
+
+### Advantages
+
+The Datadog Lambda Extension offers the following advantages over the Datadog Forwarder:
+
+- **Simple to set up**: Triggers need to be set up on the Forwarder for every new Lambda function. The Datadog Lambda extension can be easily added as a Lambda layer. Unlike with the Forwarder, configuring the Lambda extension does not require permissions to install third-party AWS CloudFormation stacks. Additionally, since the Lambda extension sends telemetry directly to Datadog, you do not need to manage CloudWatch Log group subscriptions for your Lambda functions.
+- **Less infrastructure to manage**: The Lambda extension's simpler configuration enables you to save time on infrastructure management. **Note**: You still require the Forwarder for non-Lambda AWS integrations.
+- **Skip CloudWatch Logs**: The Forwarder converts logs to metrics and traces, which are then sent to Datadog. The Datadog Lambda Extension sends traces, metrics, and logs directly to Datadog, enabling you to reduce the cost associated with CloudWatch Logs.
+
+**Note**: If you are reporting to Datadog sites in the same regions as the ones in which you are deployed (for example: if you are deployed in APAC regions and are reporting to APAC Datadog sites), your use-case is well suited for the extension. However, if you are reporting to a Datadog site in a different region than the one in which you are deployed (for example: if you are deployed in APAC regions and are reporting to US Datadog sites) the Forwarder may be a cheaper option.
```

Comment:
We can now confidently recommended the extension for these use cases. Tian addressed in this PR: https://github.com/DataDog/documentation/pull/11264